docs/application/dotnet/guides/multimedia/pose-detection.md

```diff
@@ -0,0 +1,182 @@
+LeftUpLeg,14
+LeftLowLeg,15
+LeftFoot,16
+RightUpLeg,11
+RightLowLeg,12
+RightFoot,13
+```
+If there is no mapped landmark, you don't need to list it. For example, index three is missing. It means that, in this BVH file, the third definition, `MV_INFERENCE_HUMAN_POSE_THORAX` is not defined and not used.
+
+## Prerequisites
+
+To enable your application to use the Media Vision Inference functionality:
+
+1.  Install the NuGet packages for Media Vision.
+
+    ```csharp
+    using Tizen.Multimedia;
+    using Tizen.Multimedia.Vision;
+    ```
+2. Create a structure to store pose data.
+
+   For pose detection, use the following `BodyPart` structure:
+
+   ```csharp
+   public enum BodyPart
+   {
+       Head,
+       Neck,
+       RightUpArm,
+       RightLowArm,
+       RightHand,
+       LeftUpArm,
+       LeftLowArm,
+       LeftHand,
+       RightUpLeg,
+       RightLowLeg,
+       RightFoot,
+       LeftUpLeg,
+       LeftLowLeg,
+       LeftFoot,
+   }
+   ```
+
+## Detect human pose
+
+To detect human pose from an image:
+
+1. Create the source and engine configuration handles:
+
+   ```csharp
+   pdConfig = new InferenceModelConfiguration();
+   pdConfig.WeightFilePath = Application.Current.DirectoryInfo.Resource + "model.tflite";
+   pdConfig.MetadataFilePath = Application.Current.DirectoryInfo.Resource + "model.json";
+   pdConfig.Backend = InferenceBackendType.TFLite;
+   pdConfig.Device = InferenceTargetDevice.CPU;
+   pdConfig.LoadInferenceModel();
+   ```
+
+2. Get the image file and fill the MediaVisionSource `source` with the decoded raw data.
+   In the following example image `sample.jpg`, a person is shown in a squat pose:
+
+   <img alt="sample.jpg" src="/application/native/guides/multimedia/media/mediavision_pose_sample_sumo.png" width=300>
+
+   ```csharp
+   MediaVisionSource source = new MediaVisionSource(rgbframe, width, height, Tizen.Multimedia.ColorSpace.Rgb888);
+   ```
+
+3. To detect landmarks of the pose from the `sample.jpg` image, run a `DetectAsync` from Detector.
+   This will return `Landmark`:
+
+   ```csharp
+   Task<Landmark[,]> taskLandmark = Tizen.Multimedia.Vision.PoseLandmarkDetector.DetectAsync(source, pdConfig);
+   ```
+
+4. Use `Landmark` to get landmarks information from the image:
+   Landmark is a two-dimension array that represents the number of sources for the first dimension and enum `BodyPart` for the second dimension.
+
+    ```csharp
+    public struct Landmark
+    {
+        /// <summary>
+        /// Represents a location in the 2D space.
+        /// </summary>
+        /// <since_tizen> 9 </since_tizen>
+        public Point Location
+        {
+            get;
+            set;
+        }
+        /// <summary>
+        /// Confidence score of point.
+        /// </summary>
+        /// <since_tizen> 9 </since_tizen>
+        public float Score
+        {
+            get;
+            set;
+        }
+    }
+    ```
+
+## Related information
+
+- Dependencies
+  - Tizen 6.5 or higher
```

Comment:
@KwangSon, @asm-foysal, I would recommend to do the following change for consistency sake with other Dotnet guides.

```suggestion
  - Tizen 6.5 and Higher
```

Reply:
Thanks for checking. Also please don't forget to revert the change done to dotnet/guides/multimedia/media-vision.md above.